playwright.config.ts

```diff
@@ -0,0 +1,114 @@
+import { defineConfig, devices } from "@playwright/test";
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testMatch: "**/?(*.)@(spec).*",
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : 4,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: "html",
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    headless: true,
+    baseURL: process.env.CI ? process.env.URL : "http://localhost:6006",
+
+    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+    trace: "on-first-retry",
+  },
+
+  /* Configure projects for major browsers */
+  projects: [
+    {
+      name: "chromium",
+      use: {
+        ...devices["Desktop Chrome"],
+        viewport: {
+          width: 1920,
+          height: 1080,
+        },
+      },
+    },
+    {
+      name: "chrome",
+      use: {
+        ...devices["Desktop Chrome"],
+        channel: "chrome",
+        viewport: {
+          width: 1920,
+          height: 1080,
+        },
+      },
+    },
+    {
+      name: "msedge",
+      use: {
+        ...devices["Desktop Chrome"],
+        channel: "msedge",
+        viewport: {
+          width: 1920,
+          height: 1080,
+        },
+      },
+    },
+    {
+      name: "firefox",
+      use: {
+        ...devices["Desktop Firefox"],
+        viewport: {
+          width: 1920,
+          height: 1080,
+        },
+      },
+    },
+    {
+      name: "webkit",
+      use: {
+        ...devices["Desktop Safari"],
+        viewport: {
+          width: 1920,
+          height: 1080,
+        },
+      },
+    },
+    /* Test against mobile viewports. */
+    // {
+    //   name: 'Mobile Chrome',
+    //   use: { ...devices['Pixel 5'] },
+    // },
+    // {
+    //   name: 'Mobile Safari',
+    //   use: { ...devices['iPhone 12'] },
+    // },
+    /* Test against branded browsers. */
+    // {
+    //   name: 'Microsoft Edge',
+    //   use: { ...devices['Desktop Edge'], channel: 'msedge' },
+    // },
+    // {
+    //   name: 'Google Chrome',
+    //   use: { ...devices['Desktop Chrome'], channel: 'chrome' },
+    // },
+  ],
+
+  /* Run your local dev server before starting the tests */
+  /* webServer: {
+     command: 'npm run doc',
```

Comment:
Isn't something like this faster than the 4+ minute wait for the the build+deploy of the storybook+app+nextra?

```suggestion
     command: 'npm run build:doc && npx -y http-server dist -p 6006',
```

Reply:
It is faster, but since these tests are more realistic I was thinking it may be a good idea to run them against "built code". Additionally since it finishes before the Aplitools checks the downside of taking more time is minimized.
But we can discuss it better. I might be missing something.

Reply:
They're equally "realistic", they're the "production" artifacts.
The difference is that one is served by `http-server`, the other is served by GitHub Pages' server

Reply:
Changed in new commit.